src/acceptance/administrator/components/com_users/UserNotesCest.php

```diff
@@ -0,0 +1,162 @@
+<?php
+/**
+ * @package     Joomla.Tests
+ * @subpackage  Acceptance.tests
+ *
+ * @copyright   Copyright (C) 2005 - 2018 Open Source Matters, Inc. All rights reserved.
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+namespace administrator\components\com_users;
+use Page\Acceptance\Administrator\UserNotesListPage as UserNotesList;
+use Page\Acceptance\Administrator\UserNotesFormPage as UserNotesForm;
+/**
+ * Administrator UserNotes Tests
+ *
+ * @category  Users
+ * @package   Administratorcomponentscom_Users
+ * @author    Samarth sharma <samarthsharma351@gmail.com>
+ * @copyright 2005 - 2018 Open Source Matters, Inc. All rights reserved.
+ * @license   Joomla 2005-2018
+ * @link      User Notes
+ * @since     4.0.0
+ */
+class userNotesCest
+{
+	public function __construct()
+	{
+		$this->username = "admin";
+		$this->password = "admin";
+		$this->name = "Super User";
+		$this->category = "Uncategorised";
+		$this->subject = "Test Subject";
+		$this->editorText = 'This is a test note for user '. $this->username;
```

Comment:
have an empty space here, basically something like this. 
`$this->editorText = 'This is a test note for user ' . $this->username;`

Reply:
@Samgithub1997  why is this left?